administrator-manual/en/base_system.rst

```diff
@@ -1,25 +1,94 @@
-.. _base_system-section:
+.. _legacy_server_manager-section:
 
-=================
-Base system (old)
-=================
+======================
+The old Server Manager
+======================
 
-.. note:: A new Server Manager based on Cockpit is available. See :ref:`base_system2-section`.
+This chapter describes the :dfn:`old Server Manager` web interface. Since |product| 7.9 it must be
+installed from the :ref:`software-center-section` page as it is not picked up any more by the installation
```

Comment:
... as it is not included in the default installation [procedure]

Reply:
I do not understand the change.
Can you please use the suggestion feature?